### ISSUE TITLE
ev.getX(ev.findPointerIndex(mActivePointerId)); crashes due to bad pointer index

### ISSUE THREAD

**Issue by mrhether** [sockeqwe/SwipeBack#8]
```
java.lang.IllegalArgumentException: pointerIndex out of range
       at android.view.MotionEvent.nativeGetAxisValue(MotionEvent.java)
       at android.view.MotionEvent.getX(MotionEvent.java:2014)
       at com.hannesdorfmann.swipeback.SlidingSwipeBack.onInterceptTouchEvent(SlidingSwipeBack.java:601)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:1960)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2405)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2106)
       at com.android.internal.policy.impl.PhoneWindow$DecorView.superDispatchTouchEvent(PhoneWindow.java:2369)
       at com.android.internal.policy.impl.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1719)
       at android.app.Activity.dispatchTouchEvent(Activity.java:2742)
       at com.android.internal.policy.impl.PhoneWindow$DecorView.dispatchTouchEvent(PhoneWindow.java:2330)
       at android.view.View.dispatchPointerEvent(View.java:8666)
       at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4123)
       at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:3989)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3544)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3597)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3563)
       at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:3680)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3571)
       at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:3737)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3544)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3597)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3563)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3571)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3544)
       at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:5807)
       at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:5781)
       at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:5752)
       at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:5897)
       at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:185)
       at android.os.MessageQueue.nativePollOnce(MessageQueue.java)
       at android.os.MessageQueue.next(MessageQueue.java:143)
       at android.os.Looper.loop(Looper.java:122)
       at android.app.ActivityThread.main(ActivityThread.java:5254)
       at java.lang.reflect.Method.invoke(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:372)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
```

**Comment by sockeqwe** [comment]
Thanks for reporting, I can't reproduce it yet. What is your testing device?

**Comment by mrhether** [comment]
Motorola-Moto-G-2nd-Gen-XT1063. This was through Crashalytics.

**Comment by ngocht143** [comment]
@sockeqwe: It occured when you use it to display an image in an activity. Then you use 3 or 4 fingers to zoom out and in: Keep a finger touch, use 2 or 3 fingers to zoom (in & out). The same log with @mrhether 
My testing device: Samsung Galaxy S5, Android 5.0 
Please check and fix it, looking forward to see.
Thank in advance!! :)

**Comment by sockeqwe** [comment]
Thanks for reporting, I will check that

**Comment by ngocht143** [comment]
Hi @sockeqwe: please fix this issue or simply add a try catch to avoid crash. Looking forward to see!!. Thanks :)

**Comment by sockeqwe** [comment]
Should be fixed with version `1.0.4`
